src/fable-fsharp/Replacements.fs

```diff
@@ -190,6 +193,8 @@ module Util =
                     |> makeCall com i.range (Fable.PrimitiveType (Fable.Number Int32))
                 Fable.Apply(op, [comp; makeConst 0], Fable.ApplyMeth, i.returnType, i.range)
                 |> Some
+            | Fable.Class None ->  // For enums only ("DeclaredType <enumName> Class null")
+                Fable.Apply(op, args, Fable.ApplyMeth, i.returnType, i.range) |> Some  
```

Comment:
Great! If you could work on it, to add "real" enums to Fable would be nice. 

Unfortunately my Fable-fu is not enough to do it by myself ;) But if I could help you with anything, please don't hesitate to ask me!